irohad/torii/torii_service_handler.cpp

```diff
@@ -43,7 +43,14 @@ namespace torii {
   /**
    * shuts down service handler. (actually, shuts down completion queue only)
    */
-  void ToriiServiceHandler::shutdown() { completionQueue_->Shutdown(); }
+  void ToriiServiceHandler::shutdown() { completionQueue_->Shutdown();
```

Comment:
Either use `clang-format` to format your code, or move this line with code to the next line manually.